configure.ac

```diff
@@ -180,6 +180,7 @@ AC_SUBST(LIB_ICONV)
 
 
 LIBEVENT_REQUIRED=2.0
+EVHTP_REQUIRED=1.2.3
```

Comment:
This should be 1.2.13 .

Reply:
You're right, seems I got confused in version numbers. Updated.